Makefile

```diff
@@ -1,6 +1,8 @@
-OS = $$(uname -s | tr A-Z a-z)
-BINARY_PREFIX_DIRECTORY=$(OS)_amd64_stripped
-KUBE_CONTEXT ?= minikube
+OS := $(shell uname -s | tr A-Z a-z)
+BINARY_PREFIX_DIRECTORY := $(OS)_amd64_stripped
+KUBECONTEXT ?= kubernetes-admin@kind-1
+KUBECONFIG ?= $(shell kind get kubeconfig-path)
+#KUBECONFIG ?= $$HOME/.kube/config
```

Comment:
Remove the comment and add a text comment instead referring to `kind`?
`KUBECONTEXT` is also kind-specific...